### PR TITLE
chore(release): allow to have staging step

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
avoid to promote directly releases being rolling out
it introduces an extra step to verify the release

yesterday, I manually set the release as pre-release so we could test it. Here it automates what was done.

### What issues does this PR fix or reference?

N/A

### CODE REVIEW

**Review by cdrage — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]
IMHO this is a little QE-specific. Obviously they are the primary stakeholder and approver to proceed, but this is a chance for anyone to grab the bits, do one last sanity check, or raise concerns. Ok to merge though.

**Comment by benoitf** [comment]
@deboer-tim would you mind providing a `suggested change` with a wording that fits better ?

File: RELEASE.md
Comment (deboer-tim) [inline RELEASE.md:31]: Adding some suggestions based on previous comment, feel free to not take literally or reword if you have a better idea.

`It allows to make QE tests of` =>
`It allows QE (and everyone else) to test`?

File: RELEASE.md
Comment (deboer-tim) [inline RELEASE.md:37]: `If there is red flag from QE tests,` =>
`All severe bugs and regressions are investigated and discussed. If we agree any should block the release,`

File: RELEASE.md
Comment (deboer-tim) [inline RELEASE.md:41]: `IF QE/people are giving` =>
`If committers agree we have`
(QE are committers; sounds more open source and should be a joint decision)

**Review by deboer-tim — APPROVED** [review]

**Comment by odockal** [comment]
@benoitf I appreciate this change!
@deboer-tim Totally agree with changed wording. QEs are not ultimate go-no go decision makers right now. Anyone could find a problem, QE can omit something, we can disagree about feature/bug thing, etecera.

**Comment by benoitf** [comment]
thanks @deboer-tim for the feedback

created https://github.com/containers/podman-desktop/pull/3589 with the changes

**Pull request merged**
